<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fichaje</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateEdit" name="dateSelector">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>118</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ficharButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Fichar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Exportar</string>
    </property>
   </widget>
   <widget class="QLabel" name="dayLabelCaption">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo trabajado hoy:</string>
    </property>
   </widget>
   <widget class="QLabel" name="weekLabelCaption">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo trabajado esta semana:</string>
    </property>
   </widget>
   <widget class="QLabel" name="dayLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="weekLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>372</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
